src/util/modp_atonum.c

```diff
@@ -64,24 +67,25 @@ int modp_atoi(const char *str){
 double modp_atod(const char *str){
 	
 	int negative = 0;
-	int fractionMode = 0;
+	int decimalPlaces = 0;
```

Comment:
I forgot what we agreed on for variable name format. I was assuming the general convention of camel case.